cluster-api/cloud/google/config/configtemplate.go

```diff
@@ -110,8 +110,26 @@ spec:
           limits:
             cpu: 100m
             memory: 30Mi
-      - name: controller
-        image: {{ .Image }}
+      - name: controller-manager
```

Comment:
so does this controller manager have the generic machineset controller (and will later include the generic machinedeployment one) in it and the gce-machine-controller (below) have just the gce machine controller? Is that the reason to add a third container?

Reply:
Yes. This way people can re-use our (cluster-api's) API server, MachineSet controller, MachineDeployment controller, and just replace the machine controller with whatever they need.